contracts/redeem/src/contract.rs

```diff
@@ -0,0 +1,174 @@
+// use cw2::set_contract_version;
+use crate::error::ContractError;
+use crate::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
+use crate::state::{CfgData, ADMIN, CONFIG};
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{coins, to_json_binary, Addr, BankMsg, Binary, CosmosMsg, Deps, DepsMut, Env, MessageInfo, Order, Response, StdError, StdResult};
+/*
+// version info for migration info
+const CONTRACT_NAME: &str = "crates.io:redeem";
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+*/
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn instantiate(
+    deps: DepsMut,
+    _env: Env,
+    _info: MessageInfo,
+    msg: InstantiateMsg,
+) -> Result<Response, ContractError> {
+    ADMIN.save(deps.storage, &msg.admin)?;
```

Comment:
Save the contract name and version info using cw2

https://github.com/CosmWasm/cw-minus/blob/main/packages/cw2/README.md